src/base/global.h

```diff
@@ -34,15 +34,13 @@
 const char C_TORRENT_FILE_EXTENSION[] = ".torrent";
 
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 7, 0)
 template <typename T>
-constexpr typename std::add_const<T>::type &qAsConst(T &t) noexcept { return t; }
+constexpr typename std::add_const<T>::type &asConst(T &t) noexcept { return t; }
 
 // prevent rvalue arguments:
 template <typename T>
-void qAsConst(const T &&) = delete;
-#endif
+void asConst(const T &&) = delete;
 
 // returns a const object copy
```

Comment:
It's better to say something like "Forward rvalue as const". We don't really copy it.

Reply:
Done.